L10-byteCodes/src/main/java/ru/otus/homework/Demo.java

```diff
@@ -0,0 +1,10 @@
+package main.java.ru.otus.homework;
```

Comment:
```suggestion
package ru.otus.homework;
```

Пакет для всех классов в этой папке `ru.otus.homework`, а не `main.java.ru.otus.homework`